components/kyma-environment-broker/internal/dto.go

```diff
@@ -102,6 +102,31 @@ type ProvisioningParametersDTO struct {
 type UpdatingParametersDTO struct {
 	OIDC                  *OIDCConfigDTO `json:"oidc,omitempty"`
 	RuntimeAdministrators []string       `json:"administrators,omitempty"`
+	AutoScalerMin         *int           `json:"autoScalerMin"`
+	AutoScalerMax         *int           `json:"autoScalerMax"`
+	MaxSurge              *int           `json:"maxSurge"`
+	MaxUnavailable        *int           `json:"maxUnavailable"`
+}
+
+func (u UpdatingParametersDTO) UpdateAutoScaler(p *ProvisioningParametersDTO) bool {
```

Comment:
nit: do we need the returned bool value? looks like it is not used

Reply:
It is actually used here:
https://github.com/kyma-project/control-plane/blob/393deaf9b4073b0c5e03c6727095f150d2e7952f/components/kyma-environment-broker/internal/broker/instance_update.go#L173-L175

Reply:
yes, but the returned bool value is not used

Reply:
are you planning to use it?

Reply:
it is used in the if statement I pasted above. If the functions returns true, it appends to `updateStorage`, else it skips the append